### PR TITLE
1.1 breaking changes

### PR DESCRIPTION
**Pull request by grilledham**
We can't test or merge this until 1.1, but it's good to be prepared.

### CODE REVIEW

**Comment by Jayefuu** [comment]
Glad you started this, I had meant to. I'll have a proper look later but it looks like you haven't removed our use of railgun and rail-gun dart yet.

File: locale/en/redmew_gui.cfg [hunk 1]
```diff
@@ -28,7 +28,7 @@ softmods_rank_is=Your rank is:
 softmods_market_label=Market
 softmods_market_text=On most maps you will find a market near spawn where you can use coins to\nmake purchases. Coins are acquired by chopping trees, hand crafting items and\ndestroying biter nests. Most items in the market are constant but some are\nmap-specific (usually landfill) and will rotate in and out from time to time.
 softmods_saviour_label=Train\nsavior
-softmods_saviour_text=Trains are a factorio players' worst enemy. If you have at least one small plane\nin your inventory and would be killed by a train, your life will be spared\nbut you will lose a small plane. You can get planes from the market.
+softmods_saviour_text=Trains are a factorio players' worst enemy. If you have at least one player port\nin your inventory and would be killed by a train, your life will be spared\nbut you will lose a player port. You can get planes from the market.
```
Comment (Jayefuu) [inline locale/en/redmew_gui.cfg]: Should be "Trains are a factorio players' worst enemy. If you have at least one player port\nin your inventory and are killed by a train, your life will be spared\nbut you will lose a player port. You can get player ports from the market."

**Comment by Jayefuu** [comment]
I've looked through the files and left comments. We need to have a test play and watch the logs. From Claude's small tests yesterday I found these errors in the S10 console that need looking at:
- 418.799 Script @/external/factorio/10/temp/currently-playing/utils/event_core.lua:35: Unknown ammo-category name: combat-robot-beam
- 440.032 Script @/external/factorio/10/temp/currently-playing/utils/event_core.lua:35: Unknown style icon_button
- 

Here are some things we need to check while play testing:
- Train saviour works
- Check all Redmew GUI buttons work and look visually correct. There have been lots of changes to the GUI

File: locale/en/redmew_gui.cfg (hunk 1 above)
Comment (grilledham) [reply locale/en/redmew_gui.cfg]: Should be fixed now.

**Pull request merged**